nvflare/apis/impl/controller.py

```diff
@@ -179,41 +179,37 @@ def process_task_request(self, client: Client, fl_ctx: FLContext) -> Tuple[str,
                     if client_task_to_check is None:
                         client_task_to_check = ClientTask(task=task, client=client)
                     check_status = manager.check_task_send(client_task_to_check, fl_ctx)
-                    self.logger.debug(f"checking client task {client_task_to_check} {client_task_to_check.client.name}")
```

Comment:
client_task_to_check.client.name is now no longer displayed, is that okay?

Reply:
i think that is OK, but I added it back.